src/sentry/conf/server.py

```diff
@@ -1132,7 +1139,7 @@ def create_partitioned_queues(name):
 )
 
 # Delay (in ms) to induce on API responses
-SENTRY_API_RESPONSE_DELAY = 0
+SENTRY_API_RESPONSE_DELAY = 150 if IS_DEV else None
```

Comment:
I mean, I'm obviously not going to be able to talk you out of it, and I'll just edit my own config to handle it, so it's not a big deal to me personally. Just expressing my opinion that I don't like it as a default way to assume "any Darwin machine is subject to XYZ development settings".

And, if we're going to do this, my only other opinion is to not use the environment variable `ENVIRONMENT`. I'd rather do `SENTRY_ENVIRONMENT` or something namespaced since `ENVIRONMENT` is likely to conflict with any other software.